<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://www.springframework.org/schema/beans"
	   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	   xmlns:task="http://www.springframework.org/schema/task"
	   xsi:schemaLocation="http://www.springframework.org/schema/beans
	   			https://www.springframework.org/schema/beans/spring-beans.xsd
				http://www.springframework.org/schema/task
				https://www.springframework.org/schema/task/spring-task.xsd">

	<task:annotation-driven mode="aspectj" executor="testExecutor"
							exception-handler="testExceptionHandler"/>

	<task:executor id="testExecutor"/>
<!--	<bean id="testExceptionHandler"
		  class="org.springframework.aop.interceptor.SimpleAsyncUncaughtExceptionHandler"/>-->

	<bean id="testExceptionHandler"
		  class="org.springframework.aop.interceptor.SimpleAsyncUncaughtExceptionHandler" parent="testExecutor"
		  scope="singleton">
		<description>
			my description
		</description>
		<lookup-method bean="conversionService"/>
		<replaced-method>
			<arg-type></arg-type>
		</replaced-method>
		<meta key="key" value="value"/>
		<constructor-arg>
			<bean ></bean>
		</constructor-arg>
		<qualifier value="dd"></qualifier>

	</bean>
</beans>
